docs/root/version_history/current.rst

```diff
@@ -47,6 +47,7 @@ Removed Config or Runtime
 New Features
 ------------
 * access_log: make consistent access_log format fields ``%(DOWN|DIRECT_DOWN|UP)STREAM_(LOCAL|REMOTE)_*%`` to provide all combinations of local & remote addresses for upstream & downstream connections.
+* admin: :http:post:`/logging` now accepts ``/logging?paths=name1:level1,name2:level2,...`` to change multiple log levels at once.
```

Comment:
it was my suggestion to use `':'` as a name-value separator here but I'm having second thoughts as https://en.wikipedia.org/wiki/Query_string does not call out  `':'` as a valid character for query-string values.

However https://en.wikipedia.org/wiki/Query_string?foo=bar:baz seems to pass through Chrome OK without it %-encoding the ":" so in practice it might be alright.

Another option though would be to just use more `=` characters: e.g. `/logging?paths=name1=level1,name2=level2`

wdyt?

Reply:
@jmarantz
https://datatracker.ietf.org/doc/html/rfc3986.html#appendix-A
It seems ':'(colon) is allowed in query string

Reply:
great thanks! Serves me for thinking of wikipedia as a source of truth :)